plugins/lua/sort/places.lua

```diff
@@ -111,6 +111,28 @@ local function get_location_search_key(zone)
     return table.concat(result, ' ')
 end
 
+local function get_stockpile_search_key(stockpile)
+    -- I put the stockpile_number in twice so you can search both with and without the octothorpe
+    -- (e.g. if your unnamed stockpile is listed as "Stockpile #16" you can search both "#16" or "16")
```

Comment:
is this necessary? I would have thought that a search key of `Stockpile #16` would get matched by search text of either `16` or `#16`

Reply:
You're right -- I must have forgotten to make an unnamed stockpile first before testing that the first time, or something.